utils/subset_vcf_for_phase.py

```diff
@@ -134,7 +116,10 @@ def main(args):
     )
 
     logger.info("Checkpointing sample table...")
-    sample_file_name = f"{pop}_samples{'_with' if hgdp or tgp else ''}{'_hgdp' if hgdp else ''}{'_tgp' if tgp else ''}.ht"
```

Comment:
Do you want to make an extra use case when giving a certain list of samples that are not in a pop? So you don't need AF? Because 'pop' now seems optional.

Reply:
I mentioned it in the other PR but I will make pop required.